library/net.c

```diff
@@ -714,7 +713,7 @@ int dnet_process_recv(struct dnet_backend_io *backend, struct dnet_net_state *st
 			 */
 
 			pthread_mutex_lock(&st->trans_lock);
-			dnet_trans_timestamp(st, t);
+			dnet_trans_insert_timer_nolock(st, t);
```

Comment:
Now, it doesn't update 'timeout' timestamp here. It will lead to race between io thread that will handle next response for the transaction and check thread that will find and remove timeouted transactions.

What do you think about updating 'timeout' timestamp and 'timer' tree in net threads?